server/room.ts

```diff
@@ -612,10 +613,11 @@ export class Room {
     this.io.of(this.roomId).emit('REC:removeReaction', reaction);
   };
 
-  private joinVideo = (socket: Socket) => {
+  private joinVideo = (socket: Socket, data: { isMuted: boolean }) => {
```

Comment:
Is it better to send the userMute event separately rather than changing the function here?